readme.txt

```diff
@@ -155,6 +155,9 @@ Not sure? You can find out by doing a bit a research.
 9. Membership Account page, display all sections or show specific sections using shortcode attributes.
 
 == Changelog ==
+= 2.6.1 - 2021-08-16 =
```

Comment:
We can adjust the date depending on when this goes out, it might be better to go out tomorrow and let any other things flow through support if anyone discovers anything else we'd need to fix right away

Reply:
Yea that's cool with me, we have this PR to use as a pre-release for any support requests.